src/org/pentaho/reporting/platform/plugin/output/CachingPageableHTMLOutput.java

```diff
@@ -221,6 +219,18 @@ public synchronized int generate( final MasterReport report, final int acceptedP
     }
   }
 
+  private int writeAllForScheduled( OutputStream outputStream, IReportContent cache ) throws IOException {
```

Comment:
That will produce invalid content. We are producing full HTML files, so you append multiple full HTML documents together. That is not correct. If you'd want to merge HTML documents, you'd have to do it on an XML level (ie take the contents of the body-element and put them into the merged body. But then you'd probable also have to merge the CSS. That is a totally new world of trouble.

Additionally, imagine a merged document that used 1000 pages as source. That would be so big that most browsers would struggle to display them. 

I would say keep it as separate pages as before and then we talk with Kurtis on how this part of the feature should work. Somehow I have a feeling that run-in-background for pageable HTML is not a valid option.